ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -302,6 +315,22 @@ export function useGasFeeInputs(
     hasSimulationError,
     supportsEIP1559,
     supportsEIP1559V2: supportsEIP1559 && EIP_1559_V2_ENABLED,
+    updateTransaction,
     updateTransactionUsingGasFeeEstimates,
   };
 }
+
+/**
```

Comment:
What do you think about this at the top of the file, as documentation for the hook function itself? I feel like it's likely to get missed if it's at the bottom.

Reply:
fixed